Lib/test/test_shlex.py

```diff
@@ -171,6 +171,10 @@ def testSplitPosix(self):
         """Test data splitting with posix parser"""
         self.splitTest(self.posix_data, comments=True)
 
+    def testSplitBytes(self):
+        """Test byte objects splitting"""
+        self.assertEqual(shlex.split(b"split words"), [b"split", b"words"])
```

Comment:
The coverage is insufficient. Use the `self.splitTest` as for the string case but do it for bytes inputs instead. 

In addition, use more than just ASCII characters but also those that are supported by the POSIX platforms.